### DISCUSSION TITLE
引用Natasha包之后什么也不做, 打包文件大小达20多M, 如何设置呢

### DISCUSSION THREAD

**Discussion by xibaoning** [dotnetcore/Natasha#217] [General]
如题,.NET7.0, 单文件发布,  发布后的体积太大了

**Comment by NMSAzulX** [comment]
打包大小目前是输出所有需要的程序集依赖，这是官方方案，不过下个版本有望解决。(主要是环境包输出的，不引入环境包则需要自己手动管理引用文件)

**Comment by NMSAzulX** [comment]
@xibaoning 
![image](https://github.com/dotnetcore/Natasha/assets/16236903/5ad2d134-f247-4e9a-b666-c2cfe2aa1600)
